src/Sylius/Bundle/CoreBundle/Form/Extension/CatalogPromotionRuleTypeExtension.php

```diff
@@ -5,32 +5,17 @@
 namespace Sylius\Bundle\CoreBundle\Form\Extension;
 
 use Sylius\Bundle\PromotionBundle\Form\Type\CatalogPromotionRuleType;
-use Sylius\Bundle\ResourceBundle\Form\Type\ResourceAutocompleteChoiceType;
+use Sylius\Bundle\PromotionBundle\Form\Type\CatalogRule\VariantRuleConfigurationType;
```

Comment:
```suggestion
use Sylius\Bundle\CoreBundle\Form\Type\CatalogRule\VariantRuleConfigurationType;
```
There should not be any mention about products/variants in promotion bundle